package.json

```diff
@@ -11,7 +11,7 @@
     "@actions/github": "^6.0.0",
     "@types/node": "^22.3.0",
     "husky": "^8.0.0",
-    "lint-staged": "^13.2.1",
+    "lint-staged": "^15.4.2",
```

Comment:
Just a dependency update that came up running npm audit that didn't have any breaking changes to worry about with the fix.